source/catalog.html

```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+  <body class="page__body">
+
+    <header class="main-header">
+
+      <div class="logo">
+        <ul class="logo__list">
+          <li class="logo__item">
+            <a class="logo__link" href="#">
+              <img class="logo__icon logo__icon--image" src="img/logo.svg" width="34" height="38" alt="Логотип Кэт Энерджи.">
+            </a>
+          </li>
+          <li class="logo__item">
+            <a class="logo__link" href="#">
+              <img class="logo__icon logo__icon--word" src="img/logo_word.svg" width="101" height="18" alt="Логотип с надписью Кэт Энерджи.">
+            </a>
+          </li>
+        </ul>
+      </div>
+
+      <nav class="main-nav">
+        <button class="main-nav__toogle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="index.html">
+                Главная
+              </a>
+            </li>
+            <li class="main-nav__item main-nav__item--current">
+              <a class="main-nav__link" href="#">
+                Каталог продукции
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="form.html">
+                Подбор программы
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+    </header>
+
+    <main class="catalog-main">
+
+      <div class="catalog-main__wrapper">
+        <h1 class="catalog-main__title">Каталог продукции</h1>
+        <div class="catalog-main__ list product">
```

Comment:
Тут видимо лишний пробел
```suggestion
        <div class="catalog-main__list product">
```

Reply:
Ну и вообще - этот div - лишний